pyproject.toml

```diff
@@ -90,14 +90,14 @@ source-includes = ["tests"]
 test = [
     "pytest",
     "pytest-cov",
+    "pytest-gitconfig",
     "pytest-xdist",
     "setuptools",
 ]
 
 dev = [
     "editables>=0.3",
     "pre-commit>=2.21.0",
-    "vendoring>=1.2.0; python_version ~= \"3.8\"",
```

Comment:
Why removing this

Reply:
Because it was only installed for Python 3.8 which is not supported anymore so it is not installed anymore, no ? Unless I misunderstood the line 🤔

Reply:
I restored it